source/commands.txt

```diff
@@ -149,8 +149,10 @@ Run these commands from the Command Palette to create and run
        Your playground runs against the deployment specified in your
        active connection.
        
-       |vsce| outputs the results of your playground in the
-       :guilabel:`Output` view in VS Code.
+       |vsce| splits your playground window to display the results in 
+       a pane titled :guilabel:`Playground Results.json` to the right 
+       of your playground. If you disabled split-view, |vsce| displays 
```

Comment:
[question] It's a little unclear to me as a reader how to disable split-view or what it is. Are we referring to the Editor Layout? 
<img width="748" alt="Screen Shot 2020-11-13 at 11 23 01 AM" src="https://user-images.githubusercontent.com/29379027/99095040-9d5d5000-25a2-11eb-8f39-1ddf23ed5f1e.png">

Or something different?

This question applies to the other mentions of split-view in this PR.

Reply:
Yes, this is my understanding. Updated doc.